README.md

```diff
@@ -20,12 +20,18 @@ a new file named 'foreman_custom_tab.yaml' at the location /etc/foreman/plugins/
 
 The format for your yaml file should look like the following:
 
-	---
-	  :custom_tab:
-	    :fields:
-	      IP Address, ip
-	      MAC Address, mac
-	      OS: operatingsystem.to_label
+```
+:custom_tab:
+  :fields:
```

Comment:
The existing example is wrong, but so it this update.
Please change it to:
```yaml
---
:custom_tab:
  :fields:
    IP Address: ip
    MAC Address: mac
    OS: operatingsystem.to_label
```

The keys are the display title and the values are the methods that are actually called to produce the value

Reply:
already updated in following commits